src/fu-util.c

```diff
@@ -2545,12 +2545,20 @@ fu_util_check_daemon_version (FuUtilPrivate *priv, GError **error)
 	const gchar *daemon = fwupd_client_get_daemon_version (priv->client);
 
 	if (g_strcmp0 (daemon, SOURCE_VERSION) != 0) {
-		g_set_error (error,
-			     FWUPD_ERROR,
-			     FWUPD_ERROR_NOT_SUPPORTED,
-			     /* TRANSLATORS: error message */
-			     _("Unsupported daemon version %s, client version is %s"),
-			     daemon, SOURCE_VERSION);
+		if (daemon == NULL) {
+			g_set_error_literal (error,
+					     FWUPD_ERROR,
+					     FWUPD_ERROR_NOT_SUPPORTED,
+					     /* TRANSLATORS: error message */
+					     _("Unable to connect to daemon"));
```

Comment:
maybe `daemon` is too technical and not specific enough? maybe `fwupd service` or something like that?

Reply:
I made it `service` on the logic that snap uses `snap.fwupd.fwupd.service` and regular uses `fwupd.service` and I don't want to have to show either in an error string.